dataplane/linux/endpoint_mgr_test.go

```diff
@@ -164,15 +164,19 @@ func chainsForIfaces(ifaceMetadata []string,
 	epMarkFromName := "cali-from-endpoint-mark"
 	epMarkSetOnePrefix := "cali-sm-"
 	epmarkFromPrefix := outPrefix[:6]
+	ipSetVXLANSourceHosts := "cali40all-vxlan-net"
+	ipSetAllHosts := "cali40all-hosts-net"
 	dropEncapRules := []iptables.Rule{
 		{
 			Match: iptables.Match().ProtocolNum(ProtoUDP).
+				DestIPSet(ipSetVXLANSourceHosts).
 				DestPorts(uint16(VXLANPort)),
 			Action:  iptables.DropAction{},
 			Comment: []string{"Drop VXLAN encapped packets originating in pods"},
```

Comment:
in hindsight, this comment was inaccurate, but should be correct now.

Reply:
the comment was accurate.  these rules are added to the cali interface chains so they explicitly cover packets originating in pods.  The changes is that we are now allowing vxlan packets unless they are going to the cluster VXLAN host IPs for this cluster. 

So the comment should now be something like:  "Drop VXLAN encapped packets originating in pods destined to the cluster nodes".

Reply:
Updated with tweaks for ip versions as well.